src/components/buttons/decidedbutton.module.scss

```
@@ -0,0 +1,32 @@
+@use "variables" as var;
+
+.box {
+  @apply items-center justify-center rounded-full;
+}
+
+.blue {
+  background-color: #07f;
+}
+
+.red {
+  @apply bg-gradient-to-r from-orange-500 to-red-600;
+}
+
+.aqua {
+  background-color: #0ff;
+}
+
+.god {
+  @apply bg-gradient-to-r from-cyan-500 to-blue-500;
+}
+
+.god,
+.aqua,
+.blue,
+.red {
+  @apply flex items-center justify-center rounded-full;
+}
```

html 側で`className`を適用するべきです
```suggestion
.flex-center {
  @apply flex items-center justify-center rounded-full;
}
```